lldb/test/Shell/Recognizer/verbose_trap.test

```diff
@@ -0,0 +1,9 @@
+# RUN: %clang_host -g -O0 %S/Inputs/verbose_trap.cpp -o %t.out
+# RUN: %lldb -b -s %s %t.out | FileCheck %s
+run
+# CHECK: thread #{{.*}}stop reason = Runtime Error: Function is not implemented
```

Comment:
`Runtime Error:` is pretty generic. Is there any reason we aren't mentioning that a trap was hit?

Reply:
How about `Runtime trap: ...`?